src/nmodl/config/config.h

```diff
@@ -48,6 +50,22 @@ struct NrnUnitsLib {
     /// paths where nrnunits.lib can be found
     static std::vector<std::string> NRNUNITSLIB_PATH;
 
+    static const std::string_view embedded_nrnunits;
+
+    /**
+     * Return content of units database file
+     */
+    static std::string get_content(const std::string& path) {
+        if (not path.size()) {
```

Comment:
Seems reasonable to use '!' if for no other reason than to keep SonarCloud quiet.